templates/index.html

```diff
@@ -2,8 +2,25 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta content="IE=edge" http-equiv="X-UA-Compatible">
+  <meta charset="UTF-8" />
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <!-- meta tags for better SEO -->
+  <meta name="keywords" content="FOSSASIA, meilix-generator, Meilix-Generator, meilix, Meilix,ISO,
+  custom meilix, linux, personalised linux distribution,webapp,iso" />
```

Comment:
you mention iso two times

Reply:
Also Meilix Generator and meilix. Keywords should be case insensitive.

Reply:
Comma setting is not uniform, with space or without but not both in the same line.

Reply:
@tabesin I rectifying the issues.

Reply:
I have made the comma settings uniform now.

Reply:
I deliberately mentioned "ISO" once and "iso" another time. Same is the case with meilix and Meilix Generator. I actually thought of all the keywords which a person can think off about meilix. So even if he gives the keyword of iso or ISO, meilix or Meilx search engines will show our site at the top. So, should I change them and make them case sensititve or keep them as they are now.

Reply:
Okay, I approve the PR as it is and you could fix the cosmetic issue in the next PR.